lib/rmt/downloader.rb

```diff
@@ -8,34 +8,39 @@
 require 'rmt/deduplicator'
 
 class RMT::Downloader
-  class Exception < RuntimeError
-    attr_reader :http_code, :response
-
-    def initialize(message, response: nil)
-      @response = response
-      @http_code = response&.code
-      super(message)
-    end
-  end
+  RETRIES = 4
+  RETRY_DELAY_SECONDS = 1
```

Comment:
Maybe we could keep the two second delay ?

Reply:
Done